package.json

```diff
@@ -7,18 +7,28 @@
     "jsdom": "*"
   },
   "devDependencies": {
-    "@babel/core": "^7.1.6",
-    "@babel/polyfill": "^7.0.0",
-    "@babel/preset-env": "^7.1.6",
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.4.0",
+    "@babel/plugin-proposal-class-properties": "^7.4.0",
+    "@babel/plugin-proposal-export-default-from": "^7.2.0",
+    "@babel/plugin-proposal-export-namespace-from": "^7.2.0",
+    "@babel/polyfill": "^7.4.0",
+    "@babel/preset-env": "^7.4.2",
     "@babel/register": "^7.0.0",
+    "babelify": "^10.0.0",
+    "browserify": "^16.2.3",
     "mocha": "*",
-    "mocha-jsdom": "*"
+    "mocha-jsdom": "*",
+    "python3": "0.0.1"
   },
   "scripts": {
     "test": "mocha --require @babel/register --require @babel/polyfill",
     "test-debug": "npm test -- debug -R spec",
-    "compile": "LEGO_PATH=/lib/index.js ./bin/compile.js ./demo/components/ ./demo/components.js",
-    "watch": "npm run compile -- -w"
+    "compile": "LEGO_PATH=../lib/index.js ./bin/compile.js ./demo/components/ ./demo/components.js",
+    "browserify": "browserify ./demo/components.js -o ./demo/components.js -t [ babelify --presets [ @babel/preset-env ] --plugins [ @babel/plugin-proposal-class-properties @babel/plugin-proposal-export-default-from @babel/plugin-proposal-export-namespace-from] ]",
+    "compile-past": "npm run compile && npm run browserify",
+    "watch": "npm run compile -- -w",
+    "serve": "python3 -m http.server 8080"
```

Comment:
I'm not sure if we should depend on python3 here.
This is only required to run the demo.
Eventually we could suggest in the doc to:

```
npm install http-server
http-server
open http://localhost:3000
```
WDYT?

Reply:
you right!